### ISSUE TITLE
 Please add functionality to make multi sort possible

### ISSUE THREAD

**Issue by SilviaJoycePHMS** [mbrn/material-table#116]
What is the current behavior?
I can only sort for one column. After I click on a second one, it clears the one before.

What is the use-case or motivation for changing an existing behavior?
That I can use a multiple sort based on year followed by month as well.

**Comment by Eli-Goldberg** [comment]
For us this as well this is super important.
I hope this gets implemented.

@mbrn Thanks all for the hard work, you are all great !

**Comment by mbrn** [comment]
Hi @Eli-Goldberg ,

I will apply many more features on table. But nowadays i am very busy and i can reply help-request issues in my limited time. Thanks for your patience

**Comment by Michael-Roussel** [comment]
+ 1 Multi-sort is also important for us. Great work.

**Comment by priemd** [comment]
@mbrn I needed this feature for a project and have things working great.  Wanted to get your take as there are a few ways to go with this.  Currently I added a new property called 'sortObject' to data manager which is passed onto table header through table as a prop.  It is just a simple object { myFieldA: 'asc', myFieldB: 'desc', .... } .  This really takes the place of orderBy and orderDirection, tracking all column sort state.  Wasn't sure if you prefer I just replace orderBy and orderDirection with the new 'sortObject' prop as it would handle both single and multi-sort, but might be a breaking change, need to dig a bit further.  Also, instead of a table option of 'multiSort' I've just used SHIFT key to track whether user wants current sort to be multi, if event.shiftKey is false it is single sort as usual, even if multi-sort had been active for numerous columns and user performs a new sort without holding SHIFT key then it reverts back to single sort.  Any thoughts before I take a pull?
